api/api/specs/routers.yaml

```diff
@@ -797,6 +798,9 @@ components:
                 description: "route id of the routes configured in the router"
                 type: "string"
                 example: "route-1"
+        route_name_path:
+          type: "string"
+          pattern: '^\w*(?:\.\w+)*$'
```

Comment:
This regex pattern here allows only period-delimited words of letters (or underscores) to be accepted as a valid `route_name_path`. Specifying this here also dispenses the need for manually crafting regex checks on the API for this field.

Reply:
Oooh what if the nested path actually has a key with "-" in it? Shall we remove this check and keep it flexible like the `TrafficRuleCondition` ?

Reply:
Hmm if it's just dashes we can make it `^[\w\-]*(?:\.[\w\-]+)*$` instead? Are you referring to `field` within `TrafficRuleCondition`? That one doesn't have any checks right?

Reply:
Okay as discussed earlier, I'll be removing the regex changes to allow users more flexibility (at their own discretion) at specifying the route path names. Any resultant errors from any incorrect parsing of the path would be returned by the router itself.